### PR TITLE
test: add test cases to confirm output for `PinnedCodes`

### PR DESCRIPTION
**Pull request by kokeshiM0chi**
### PinnedCodes: TestQueryPinnedCodes
Add a test case because we have not tested for pagination.

```
message QueryPinnedCodesResponse {
  repeated uint64 code_ids = 1 [ (gogoproto.customname) = "CodeIDs" ];
  // pagination defines the pagination in the response.
  cosmos.base.query.v1beta1.PageResponse pagination = 2;
}
```

spec:
https://github.com/CosmWasm/wasmd/blob/main/proto/cosmwasm/wasm/v1/query.proto#L236

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/wasmd/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/wasmd/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have added a relevant changelog to `CHANGELOG.md` 
- [x] I have added tests to cover my changes. 
- [ ] I have updated the documentation accordingly. (not needed)
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml` (not needed)

### CODE REVIEW

File: x/wasm/keeper/querier_test.go [hunk 1]
```diff
@@ -708,17 +722,22 @@ func TestQueryPinnedCodes(t *testing.T) {
...
 			got, err := q.PinnedCodes(sdk.WrapSDKContext(ctx), spec.srcQuery)
-			if spec.expErr != nil {
-				assert.Nil(t, got)
-				assert.NotNil(t, err)
 
+			if testName == "req nil" || testName == "with invalid pagination key" {
+				assert.Nil(t, got)
+				assert.EqualError(t, spec.expErr, err.Error())
 				return
 			}
```
Comment (170210) [inline x/wasm/keeper/querier_test.go:733]:
```suggestion
			if spec.expErr != nil {
				assert.Nil(t, got)
				assert.EqualError(t, spec.expErr, err.Error())
			} else {
			}
```

Comment (170210) [reply x/wasm/keeper/querier_test.go:733]: I think we need to standardize the use of assertion errors, but that will be implemented in other PRs.

**Review by 170210 — COMMENTED** [review]
Please add changelog in CHANGELOGS.md

File: x/wasm/keeper/querier_test.go [hunk 2]
```diff
@@ -708,17 +722,22 @@ func TestQueryPinnedCodes(t *testing.T) {
 			expCodeIDs: []uint64{exampleContract2.CodeID},
 		},
 	}
-	for msg, spec := range specs {
-		t.Run(msg, func(t *testing.T) {
+	for testName, spec := range specs {
+		t.Run(testName, func(t *testing.T) {
```
Comment (loloicci) [inline x/wasm/keeper/querier_test.go:726]: is this change needed?

File: x/wasm/keeper/querier_test.go [hunk 3]
```diff
@@ -708,17 +722,22 @@ func TestQueryPinnedCodes(t *testing.T) {
...
 			got, err := q.PinnedCodes(sdk.WrapSDKContext(ctx), spec.srcQuery)
+
 			if spec.expErr != nil {
 				assert.Nil(t, got)
-				assert.NotNil(t, err)
-
+				assert.EqualError(t, spec.expErr, err.Error())
 				return
-			}
+			}
```
Comment (170210) [inline x/wasm/keeper/querier_test.go:733]: Sorry, it seems that I had add one space after } in the before comment, please check it.

File: x/wasm/keeper/querier_test.go (hunk 2 above)
Comment (kokeshiM0chi) [reply x/wasm/keeper/querier_test.go:726]: Unnecessary change. Revert changes. Thanks.

File: x/wasm/keeper/querier_test.go (hunk 3 above)
Comment (kokeshiM0chi) [reply x/wasm/keeper/querier_test.go:733]: Thanks for telling me! I removed it!

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,10 +6,12 @@
 * [\#61](https://github.com/Finschia/wasmd/pull/61) bumpup ibc-go to v4
 
 ### Improvements
+* [\#64](https://github.com/Finschia/wasmd/pull/64) test: add test cases to confirm output for PinnedCodes
 * [\#63](https://github.com/Finschia/wasmd/pull/63) add event checking to TestStoreCode
 * [\#65](https://github.com/Finschia/wasmd/pull/65) add test cases for empty request in each function
 * [\#66](https://github.com/Finschia/wasmd/pull/66) add test cases for invalid pagination key in some functions
 
+
```
Comment (170210) [inline CHANGELOG.md:14]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,10 +6,12 @@
 * [\#61](https://github.com/Finschia/wasmd/pull/61) bumpup ibc-go to v4
 
 ### Improvements
+* [\#64](https://github.com/Finschia/wasmd/pull/64) test: add test cases to confirm output for PinnedCodes
 * [\#63](https://github.com/Finschia/wasmd/pull/63) add event checking to TestStoreCode
 * [\#65](https://github.com/Finschia/wasmd/pull/65) add test cases for empty request in each function
 * [\#66](https://github.com/Finschia/wasmd/pull/66) add test cases for invalid pagination key in some functions
```
Comment (170210) [inline CHANGELOG.md:12]: I think it's better to follow the order of merging.

File: x/wasm/keeper/querier_test.go [hunk 6]
```diff
@@ -736,19 +736,28 @@ func TestQueryPinnedCodes(t *testing.T) {
 
 	q := Querier(keeper)
 	specs := map[string]struct {
-		srcQuery   *types.QueryPinnedCodesRequest
-		expCodeIDs []uint64
-		expErr     error
+		paginationTotal         uint64
+		paginationNextKeyLength uint64
+		codeIdsLength           uint64
```
Comment (170210) [inline x/wasm/keeper/querier_test.go:741]: I think the codeIdsLength is duplicated with expCodeIDs. If codeIdsLength is wrong, expCodeIDs must not be equal.

File: CHANGELOG.md (hunk 4 above)
Comment (kokeshiM0chi) [reply CHANGELOG.md:13]: This blank line before `###` is needed, isn't it?

Comment (170210) [reply CHANGELOG.md:13]: Yes, but it seems you have two blank lines before.

**Review by 170210 — DISMISSED** [review]
LGTM

File: CHANGELOG.md (hunk 5 above)
Comment (kokeshiM0chi) [reply CHANGELOG.md]: I fixed it.

**Review by 170210 — APPROVED** [review]

**Review by da1suk8 — APPROVED** [review]
